opensearch_py_ml/ml_commons/ml_commons_client.py

```diff
@@ -63,6 +63,71 @@ def upload_model(
             model_path, model_config_path, isVerbose
         )
 
+    def upload_pretrained_model(self, name: str, version: str, model_format):
```

Comment:
So, if I'm not mistaken, to upload a pretrained model, we have to:

1. Send model info to /models/_upload
2. Then load the model using load model api, after which ML Commons downloads the model chunks from Hugging Face by itself.

If I do the above 2 steps, I can successfully use the model for inference in dev tools.

Are you saying that that I shouldn't do the second part? Because without it, if I try to do inference, I get the "model not loaded" error. 

I would appreciate it if you clarified this for me.

Reply:
I think it's a great idea. The argument was added.

Also, about sending model info to upload api, that's what _create_model_metadoc method does anyway. I just misnamed the method.

Both of these issues were fixed in the last commit.